gateway/api/management/commands/free_resources.py

```diff
@@ -20,13 +20,20 @@ class Command(BaseCommand):
 
     def handle(self, *args, **options):
         compute_resources = ComputeResource.objects.filter(active=True)
-        counter = 0
 
         for compute_resource in compute_resources:
+            # I think this logic could be reviewed because now each job
+            # would have its own compute resource but let's do that
+            # in an additional iteration
             alive_jobs = Job.objects.filter(
                 status__in=Job.RUNNING_STATUSES, compute_resource=compute_resource
             )
```

Comment:
```suggestion
            # I think this logic could be reviewed because now each job
            # would have its own compute resource but let's do that
            # in an additional iteration
            are_there_alive_jobs = Job.objects.filter(
                status__in=Job.RUNNING_STATUSES, compute_resource=compute_resource
            ).exists()
```
Make sense, but if we cannot remove it, we can do a `.count()` or `.exists()`, it is probably faster than retrieving all jobs and we are only using `len(alive_jobs)` some lines below this.

Reply:
Done [f160ab8](https://github.com/Qiskit/qiskit-serverless/pull/1749/commits/f160ab80c393c012314ac342da023ad161b649a8)!